ticketbuyer/tb.go

```diff
@@ -60,9 +60,9 @@ type TB struct {
 	mu  sync.Mutex
 }
 
-// New returns a new TB to buy tickets from a wallet using the default config.
-func New(w *wallet.Wallet) *TB {
-	return &TB{wallet: w}
+// New returns a new TB to buy tickets from a wallet.
+func New(cfg Config, w *wallet.Wallet) *TB {
```

Comment:
please reverse these args

`func New(w *wallet.Wallet, cfg Config) *TB`

reads better when calling it with a config literal:

```
			tb := ticketbuyer.New(w, ticketbuyer.Config{
				BuyTickets:         cfg.EnableTicketBuyer,
				Account:            purchaseAccount,
				Maintain:           cfg.TBOpts.BalanceToMaintainAbsolute.Amount,
				VotingAddr:         votingAddr,
				Limit:              int(cfg.TBOpts.Limit),
				VotingAccount:      votingAccount,
				Mixing:             cfg.Mixing,
				MixChange:          cfg.MixChange,
				MixedAccount:       mixedAccount,
				MixedAccountBranch: cfg.mixedBranch,
				TicketSplitAccount: ticketSplitAccount,
				ChangeAccount:      changeAccount,
				VSP:                vspClient,
			})
```